### PR TITLE
for the failcount check of the resources switch from 'crm_mon' to 'ci…

### PR DESCRIPTION
**Pull request by angelabriel**
…badmin' to prevent problems, if 'crm_mon' output format will change

### CODE REVIEW

File: SAPHana/bin/SAPHanaSR-manageAttr [hunk 1]
```diff
@@ -999,9 +999,13 @@ chk_cluster_health() {
...
     fi
-    #rs=$(crm_mon -r1 2>/dev/null | awk '/Clone Set:/ { print $3 }')
+
+    # rs=$(crm_mon -r1 2>/dev/null | awk '/Clone Set:/ { print $3 }')
     # 'Clone Set:' or 'Master/Slave Set:' (on older pacemaker-cli versions)
-    rs=$(crm_mon -r1 2>/dev/null | awk '/[A-Za-z/ ][[:space:]]Set:/ { print $3 }')
+    # rs=$(crm_mon -r1 2>/dev/null | awk '/[A-Za-z/ ][[:space:]]Set:/ { print $3 }')
+    # if crm_mon output will change, awk's 'print $3' will no longer work correctly
+    # switch to cibadmin and xmllint instead.
+    rs=$(cibadmin -Q | xmllint -xpath "(//*[primitive[@*]]/@id)" - | sed -e 's/id=//g' -e 's/"//g')
```
Comment (fmherschel) [inline SAPHana/bin/SAPHanaSR-manageAttr:1008]: @angelabriel The idea is to serach for objects containing a primitive which have set at least one attribute and to return the object id. If you like to filter-out other clones or multi-state resources you might filter by the resource types:
`cibadmin -Q | xmllint -xpath '(//*[primitive[@type="SAPHanaTopology" or @type="SAPHanaController"]]/@id)' -`

Comment (angelabriel) [reply SAPHana/bin/SAPHanaSR-manageAttr:1008]: The origin idea from Lars was to check all clone or multi-stat resources for failcounts and warn the admin. But we can change this behaviour and shrink the check to 'our' resources. No problem, needs only a decision :-)

Comment (fmherschel) [reply SAPHana/bin/SAPHanaSR-manageAttr:1008]: So we release the PR for now and discuss that after vacation period ;-)

**Review by fmherschel — APPROVED** [review]
Looks good for me . To be discussed later, if we like to filter that more to "our" clone and multi-state resources.

**Pull request merged**